lib/template.js

```diff
@@ -52,5 +52,33 @@ function validateReplacers({ context, replacers }) {
     .filter(Boolean)
 }
 
+function validateAutolabeler({ context, autolabeler }) {
+  return autolabeler
+    .map((autolabel) => {
+      try {
+        return {
+          ...autolabel,
+          branch: autolabel.branch.map((reg) => {
+            return toRegex(reg)
+          }),
+          title: autolabel.title.map((reg) => {
+            return toRegex(reg)
+          }),
+          body: autolabel.body.map((reg) => {
+            return toRegex(reg)
+          }),
+        }
+      } catch (e) {
+        console.log({
```

Comment:
```suggestion
        log({
```

I think your habit here turned to console.log rather than the built in log

Reply:
Yes. I'm forward-porting these from an old fork, so mistakes happen :)